### PR TITLE
[dimo-erc20-votes-vesting] Dimo erc20 votes vesting

### PR DESCRIPTION
**Pull request by kostyamospan**
Changes proposed in this pull request:
- strategy for dimo project

### CODE REVIEW

File: src/strategies/dimo-erc20-votes-vesting/index.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        snapshot
+    );
+
+    const multi = new Multicaller(network, provider, abi, { blockTag });
+    addresses.forEach((address) =>
+        multi.call(address, options.vesting, 'balanceOf', [address])
+    );
+
+    const vestingScore: Record<string, BigNumberish> = await multi.execute();
+
```
Comment (ChaituVR) [inline src/strategies/dimo-erc20-votes-vesting/index.ts:38]: Hi @kostyamospan You can use two `erc20-balance-of` strategies for this

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/dimo-erc20-votes-vesting/index.ts (hunk 1 above)
Comment (borseno) [reply src/strategies/dimo-erc20-votes-vesting/index.ts:38]: Hello! Where is it possible to use two strategies? Could you send a screenshot of UI, where it's located (where I can use two strategies)? I couldn't find it anywhere.

Comment (ChaituVR) [reply src/strategies/dimo-erc20-votes-vesting/index.ts:38]: Hi @borseno you can set this in your space settings, under strategies add two `erc20-balance-of` strategies

Comment (ChaituVR) [reply src/strategies/dimo-erc20-votes-vesting/index.ts:38]: image

File: src/strategies/dimo-erc20-votes-balanceOf/index.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    );
+
+    return Object.fromEntries(
+        addresses.map((address) => {
+            const addressScoreTokenVotes = erc20votesScore[address] ?? 0;
+            const addressContractScore = balanceOfScore[address] ?? 0;
+
+            return [
+                address,
+                addressContractScore + addressScoreTokenVotes
```
Comment (ChaituVR) [inline src/strategies/dimo-erc20-votes-balanceOf/index.ts:52]: Hi @kostyamospan you can select two strategies (erc20-votes and erc20-balance-of) from your space settings

Comment (ChaituVR) [reply src/strategies/dimo-erc20-votes-balanceOf/index.ts:52]: Please let me know if it is not what you are looking for

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ChaituVR** [comment]
Closing this as stale, feel free to reopen if needed

**Pull request closed without merging**